accepted/2.3/spread-collections/benchmarks/bin/profile.dart

```diff
@@ -281,9 +281,9 @@ void iterableIterator(Iterable<String> from, List<String> to) {
 }
 
 void iterableForEach(Iterable<String> from, List<String> to) {
-  var temp = to;
+  List<String>? temp = to;
   from.forEach((s) {
-    temp.add(s);
+    temp!.add(s);
```

Comment:
Same here, move it to `= to!;`

Reply:
ACK. Not great code to begin with. 
`from.forEach(to!.add);` would be better, but even that's really just `to!.addAll(from)`.